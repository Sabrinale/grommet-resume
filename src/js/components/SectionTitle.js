import React from 'react'

import { Heading } from 'grommet'

export const SectionTitle = ({ children }) => (
  <Heading
    level={3}
    margin={{ top: 'xxsmall', bottom: 'xsmall', left: '1px' }}
    pad={{ top: 'xsmall', bottom: 'xsmall' }}
  >
    {children}
  </Heading>
)
