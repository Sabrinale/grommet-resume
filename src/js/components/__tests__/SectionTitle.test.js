import React from 'react'
import renderer from 'react-test-renderer'

import { SectionTitle } from '../SectionTitle'

test('renders section title element', () => {
  const component = renderer.create(<SectionTitle>test</SectionTitle>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
