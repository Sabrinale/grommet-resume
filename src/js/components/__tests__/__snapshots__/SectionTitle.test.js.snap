// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders section title element 1`] = `
<h3
  className="StyledHeading-sc-1rdh4aw-0 jVEgcZ"
>
  test
</h3>
`;
